cli/endpoints/online/mlflow-vnet/environment/docker-context.yml

```diff
@@ -0,0 +1,13 @@
+$schema: https://azuremlschemas.azureedge.net/latest/environment.schema.json
+name: vnet-400
+image: 7acfa0544d574b78b2d66ed4939ef465.azurecr.io/mlflowbyoc:1
```

Comment:
@shivanissambare - the acr here should be a reference (Let's refer byoc R sample)